.mailmap

```diff
@@ -24,3 +24,4 @@ Aaron Cole            <aaron@epiphron.co.nz>
 Jamie Allen           <jamie.allen@metoffice.gov.uk>           JAllen42                <jrallen42@gmail.com>
 Jamie Allen           <69790602+JAllen42@users.noreply.github.com>
 Christopher Bennett   <christopher.bennett@metoffice.gov.uk>   ChrisPaulBennett
+Mark Dawson           <mark.dawson@metoffice.gov.uk>       Mark Dawson         <mark.dawson@metoffice.gov.ukpython>
```

Comment:
Not important, but I'm curious how "metoffice.gov.ukpython" arose?

Reply:
Im not sure but I think its everywhere 😔

Reply:
Check your `~/.gitconfig`